packages/algorand-js/src/interpreter/interpreter.ts

```diff
@@ -19,7 +19,9 @@ export class Interpreter {
   tx: Txn;
   gtxs: Txn[];
   accounts: Map<string, AccountState>;
+  accountsAssets: Map<string, Map<number, AccountAssetInfo>>;
   globalApps: Map<number, SSCParams>;
+  globalAssets: Map<number, AssetParams>;
```

Comment:
For the moment this is fine. But we will need to move it to a `ctx` -> I was writing about this in @ratik21  PR. Please coordinate for the future PR to update it (to avoid conflicts).

Reply:
Ok